src/app/clusters/diagnostic-logs-server/diagnostic-logs-server.cpp

```diff
@@ -160,37 +162,57 @@ void DiagnosticLogsServer::HandleLogRequestForBdx(CommandHandler * commandObj, c
 #endif // CHIP_CONFIG_ENABLE_BDX_LOG_TRANSFER
 }
 
-} // namespace DiagnosticLogs
-} // namespace Clusters
-} // namespace app
-} // namespace chip
+void DiagnosticLogsServer::InvokeCommand(HandlerContext & handlerContext)
+{
+    switch (handlerContext.mRequestPath.mCommandId)
+    {
+#ifdef MATTER_DM_DIAGNOSTIC_LOGS_CLUSTER_CLIENT_ENDPOINT_COUNT
```

Comment:
Why do we care about client endpoints?

Reply:
This is a test, and it should be DIAGNOSTIC_LOGS_ENABLE_RETRIEVE_LOGS_REQUEST_CMD